crates/vm2/src/instruction_handlers/common.rs

```diff
@@ -1,36 +1,46 @@
 use zksync_vm2_interface::{opcodes, OpcodeType, Tracer};
 
 use super::ret::free_panic;
-use crate::{addressing_modes::Arguments, instruction::ExecutionStatus, VirtualMachine};
+use crate::{addressing_modes::Arguments, instruction::ExecutionStatus, VirtualMachine, World};
 
 #[inline(always)]
-pub(crate) fn boilerplate<Opcode: OpcodeType, T: Tracer, W>(
+pub(crate) fn boilerplate<Opcode, T, W>(
```

Comment:
IMO where clauses are good when types don't fit onto one line or when they are necessary. Do you or clippy disagree?

Reply:
No strong opinion from either me or clippy, so I'll revert it back.